End-to-End-Tests.md

```diff
@@ -167,10 +167,16 @@ Extensions provide `webdriverio.js` files to make them easier to test. The E2E t
 
 If you don't know the name of the suite you want to run, you can find it in `core/tests/wdio.conf.js`. Then you can run your test like this:
 
+Python:
 ```console
 python -m scripts.run_e2e_tests --suite="suiteName"
 ```
 
+Docker:
+```console
+make run_tests.e2e PYTHON_ARGS="--suite=suiteName"
```

Comment:
throwing error

Reply:
Try this https://stackoverflow.com/a/32600959
It should fix the issue

Reply:
> Try this https://stackoverflow.com/a/32600959 It should fix the issue

yeah after changing the max user watches working fine